lib/sqs_pusher.rb

```diff
@@ -0,0 +1,19 @@
+require 'aws-sdk'
+
+
+class SqsPusher
+
+  def self.push(params)
+
+    sqs_client = Aws::SQS::Client.new
+    queue_name = 'post_test'
+    queue_url = sqs_client.get_queue_url({queue_name:queue_name}).queue_url
+
+    sqs_client.send_message({
+       queue_url: queue_url, # required
+       message_body: params.to_json, # required
+    })
+
+  end
+
+end
```

Comment:
`client` in `push` 
```ruby
def push
  client.send_message(
    queue_url: queue_url,
    message_body: @params.to_json,
  )
end
```
will need to be `ChampaignQueue.client`, right? Same as in `queue_url`? Why not just specify `@client` as `ChampaignQueue.client` in `initialize`?